website/src/components/Footer.tsx

```diff
@@ -30,7 +30,7 @@ export const Footer: React.FunctionComponent<{ minimal?: boolean }> = ({ minimal
                                         rel="noopener"
                                         className="bigCode"
                                     >
-                                        Not a developer?
+                                        Sourcegraph Overview (PDF)
```

Comment:
```suggestion
                                        Sourcegraph overview (PDF)
```

Using sentence case as per the guidelines in https://about.sourcegraph.com/handbook/communication/content_guidelines/actionable_language#headings-and-subheadings.

Reply:
fixed